File: backend/wordsAPI.py
import http.client

conn = http.client.HTTPSConnection("wordsapivi.p.rapidapi.com")

headers = {
    'x-rapidapi-key': "ce19d0164fmsh3d383efc0e85ce5p16dcb1jsnb1a4a3c79541",
    'x-rapidapi-host': "wordsapiv1.p.rapidapi.com"
}

conn.request("GET", "/words/%7Bword%7D", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: src/components/wordsAPI.ts
import axios from 'axios';

const options = {
    method: 'GET',
    url: 'https://wordsapiv1.p.rapidapi.com/words/',
    headers: {
        'x-rapidapi-key': '2b74b698b7msh54cf0f18eeff0c2p1b1cf1jsn257225d28e8e',
        'x-rapidapi-host': 'wordsapiv1.p.rapidapi.com'
    }
};

try {
	const response = await axios.request(options);
	console.log(response.data);
} catch (error) {
	console.error(error);
}


